RecoHI/HiCentralityAlgos/python/HiCentrality_cfi.py

```diff
@@ -36,11 +36,18 @@
 
 from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
-for e in [pp_on_XeXe_2017, pp_on_AA_2018]:
-    e.toModify(hiCentrality,
+
+for p in [pp_on_AA_2018]:
+    p.toModify(hiCentrality,
                producePixelTracks = True,
                srcPixelTracks = "hiConformalPixelTracks",
-               srcTracks = cms.InputTag("generalTracks"),
-               srcVertex = cms.InputTag("offlinePrimaryVertices")
+               srcTracks = "generalTracks",
+               srcVertex = "offlinePrimaryVertices"
                )
 
+for e in [pp_on_XeXe_2017]:
+    e.toModify(hiCentrality,
+               producePixelTracks = False,
```

Comment:
@abaty @mandrenguyen 
shouldn't we instead add hiConformalPixelTracks to XeXe reco version?

Reply:
More specifically, I mean here 
https://github.com/cms-sw/cmssw/blob/ab2e23a109cb27fa971d681415be121835684b35/Configuration/StandardSequences/python/Reconstruction_cff.py#L163-L167
to have ```(pp_on_XeXe_2017 | pp_on_AA_2018).```

Reply:
@slava77 I don't see a motivation to add this to XeXe.  I don't know of any plan to reprocess that data.  But I don't care either way.

Reply:
the goal is to make a more uniform support for pp_on_ configurations and hopefully even make just one.

Reply:
> @slava77 @mandrenguyen so what is the preferred option?

my preferred solution is to have hi pixel tracks in XeXe in the same way as pp_on_AA
so that we do not go back to this again with another possible addition to pp_on_AA using these tracks (think of DQM etc).

Reply:
Indeed we should just make this a single era for pp_on_AA.  Probably right this second is not the right time to do that.  Perhaps an era cleanup is in order for 10_4_X.  If i promise to undertake that in December, can we just go with the current with the current PR as is?  I'm not sure this is worth holding things up.

Reply:
@slava77 Fine.   For the master, I can just submit a PR adding the conformal pixel tracks to XeXe.  For 10_3, the changes in this PR have already been incorporated into #24889  which is still pending.   How should we handle that?

Reply:
@mandrenguyen What about reverting the last commit in #24889, and adding the new one (which needs to be evaluated and merged in the master fisrt, anyhow)?